skfem/mesh/mesh.py

```diff
@@ -1316,12 +1316,14 @@ def param(self):
 
     def element_finder(self, mapping=None):
         ix = np.argsort(self.p)
+        mint = self.t[np.argmin(self.p[:, self.t], axis=1)[0],
```

Comment:
We could replace `np.argmin(self.p[:, self.t], axis=1)[0]` with
```python
np.argmin(self.p[0, self.t], axis=1)
```
but this line might need changing anyway.